src/interface/interface.ts

```diff
@@ -46,13 +46,13 @@ export interface Task {
   md5: string;
   task_type: string;
   status: TaskStatus;
-  single_proof: Uint8Array;
-  proof: Uint8Array;
+  single_proof: Array<Uint8Array>;
```

Comment:
As we changed it so anyway the interface will change so it is a good time we rename them to matching.
So currently, single proof and instances are for single proof.
proof and batch_instances are for aggr proof.

It is not good. We should rename them to 
single_proofs and single_instances
batch_proofs and batch_instances

Reply:
Also please update the examples/src/queries/task.rs as we updated and it need change.

Reply:
Maybe need wait for a moment as we may do not want to change this interface for UI and client.
This is because in fact it need change the examples too, also the important thing is the verify method.
For example, if we review the examples/src, we can see not only query but also VerifyProof files need update. This may affect our business client.

I will check with Xin